### ISSUE TITLE
Please add support for AndroidTV as well

### ISSUE THREAD

**Issue by nitantsoni** [martinmarinov/AndroidDvbDriver#10]
I am using Nougat on my Odroid XU4 and it is working great with an AndroidTV Image, though this package is not available in the TV version. Could you please add it for that too?

**Comment by rozaimialli** [comment]
Test

**Comment by salehdeh76** [comment]
Did you find anything useful? As far as I perceived, this driver works just fine with TV but someone should write a player app (using this driver) for it. If there is no such app yet, I am ready (as a volunteer developer) to start such a project! That should be very nice to have an ATV app for this driver.

**Comment by nitantsoni** [comment]
Nope, gave up on ATV a long while back. Ended up using Debian + TVHeadEnd to get DVR working.
